dova/www/js/controllers.js

```diff
@@ -1,8 +1,13 @@
 angular.module('app.controllers', [])
   
 .controller('signupCtrl', ['$scope', '$stateParams', '$http', '$state', '$rootScope',
+    
 function ($scope, $stateParams, $http, $state, $rootScope) {
+    $scope.offline_debug = false;
```

Comment:
I tried to implement a global var, but it that would require me to start a service and send that service to any controller I wanna debug, this is easier for me to handle for now, all the debugging stuff will be taken out before ending the project